stdlib/http/src/main/ballerina/src/http/http_connection.bal

```diff
@@ -172,13 +172,26 @@ public type Caller client object {
         response.statusCode = STATUS_ACCEPTED;
         return self->respond(response);
     }
+
+    # Gets the hostname from the remote address. This method may trigger a DNS reverse lookup if the address was created
+    # with a literal IP address.
+    #
+    # + return - The hostname of the address or `nil` if it is unresolved
```

Comment:
```suggestion
    # + return - The hostname of the address or else `()` if it is unresolved
```

Reply:
It is fixed in https://github.com/ballerina-platform/ballerina-lang/pull/22779/files#diff-c90efb208c957133fd64332cd26e00f1R241